### PR TITLE
chore: [PLAT-67] switch to using the jaeger grpc exporter

### PR DESCRIPTION
**Pull request by jkwatson-verta**
## Testing
- [x] Unit test
- [x] Deployed to dev env
- [ ] Other (explain) 

## Reverting
- [ ] Contains Migration - _Do Not Revert_

### CODE REVIEW

**Review by conradoverta — APPROVED** [review]

File: backend/common/pom.xml [hunk 1]
```diff
@@ -109,6 +113,10 @@
             <groupId>io.opentelemetry</groupId>
             <artifactId>opentelemetry-exporter-jaeger-thrift</artifactId>
         </dependency>
+        <dependency>
```
Comment (liam-verta) [inline backend/common/pom.xml]: Should we removed the `opentelemetry-exporter-jaeger-thrift` above?

**Comment by liam-verta** [comment]
You'll also need to update the downstream projects right?

File: backend/common/pom.xml (hunk 1 above)
Comment (jkwatson-verta) [reply backend/common/pom.xml]: yes! good call.

**Comment by jkwatson-verta** [comment]
> You'll also need to update the downstream projects right?

I think now that this code all lives here, that it should be ok? Unless we aren't properly depending on the common module in them?

File: backend/common/pom.xml (hunk 1 above)
Comment (jkwatson-verta) [reply backend/common/pom.xml]: done

**Comment by jkwatson-verta** [comment]
> > You'll also need to update the downstream projects right?
> 
> I think now that this code all lives here, that it should be ok? Unless we aren't properly depending on the common module in them?

unfortunately, it does look like we'll need a couple of updates to the dependencies downstream, yes. I'll jump on them ASAP after this merges.

**Review by coryverta — APPROVED** [review]

**Pull request merged**